Package.swift

```diff
@@ -79,6 +83,8 @@ let SwiftFirebase =
                           "-Lthird_party/firebase-development/usr/libs/windows",
                           "-Lthird_party/firebase-development/usr/libs/windows/deps/app",
                           "-Lthird_party/firebase-development/usr/libs/windows/deps/app/external",
+                          "-Xlinker",
+                          "-ignore:4217"
```

Comment:
We ignore this warning in other projects and it was causing a lot of chatter in the build logs so I figured it was okay to ignore here.

Reply:
I really would rather that we not do this, especially as if we fix the issue with SPM, there are still potentially cases of miscompilation that this can identify.

Reply:
Removed the ignore directive here.